### DISCUSSION TITLE
No sources found, Playback error, doesnt scrape.

### DISCUSSION THREAD

**Discussion by areebhussain321** [SwagOtaku/plugin.video.kaito#54] [General]
Doesnt work on Kodi at all. Was fine a few days ago. All my other friends have this issue. NYAA isnt working neither is GOGO. Really unhappy with this wishing for a fix hopefully. Sometimes it does work however the quality would be very bad, for example im trying to watch Attack On Titan and only one 480p source would come up and thats it compared to a week ago id get like 10+ sources and theyd mostly be 1080p. ive tried trouble shooting and done everything so just a problem on your side. 

edit: found a fix and worked. https://www.reddit.com/r/Addons4Kodi/comments/m9o6ph/a_solution_for_kaito/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by ctanom** [comment]
Same issue  as of 3/22 
Was working literally yesterday or day before
Tried reauth debrid, uninstalling and reinstalling, complete uninstall of kodi and reinstall only this addon still nothing. :(

**Reply by areebhussain321** [reply]
Try this. Worked for me

https://www.reddit.com/r/Addons4Kodi/comments/m9o6ph/a_solution_for_kaito/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by ctanom** [comment]
You ever get this resolved?

> Doesnt work on Kodi at all. Was fine a few days ago. All my other friends have this issue. NYAA isnt working neither is GOGO. Really unhappy with this wishing for a fix hopefully. Sometimes it does work however the quality would be very bad, for example im trying to watch Attack On Titan and only one 480p source would come up and thats it compared to a week ago id get like 10+ sources and theyd mostly be 1080p. ive tried trouble shooting and done everything so just a problem on your side.
> 
> Im on kodi 18.8/9 btw, doesnt work on any version of Kodi

**Reply by areebhussain321** [reply]
Yes. Found a fix. Worked for me.

https://www.reddit.com/r/Addons4Kodi/comments/m9o6ph/a_solution_for_kaito/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by areebhussain321** [comment]
Found a fix 🙂

https://www.reddit.com/r/Addons4Kodi/comments/m9o6ph/a_solution_for_kaito/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by ctanom** [comment]
Thank you I appreciate it so much! Weird how that one little thing crippled
it like that
